pkgs/applications/networking/p2p/libutp/3.4.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, lib, fetchFromGitHub, cmake, unstableGitUpdater }:
+
+stdenv.mkDerivation rec {
+  pname = "libutp";
+  version = "unstable-2017-01-02";
```

Comment:
This date is incorrect https://github.com/transmission/libutp/commit/9cb9f9c4f0073d78b08d6542cebaea6564ecadfe

Reply:
Indeed, forgot to update this. Fixed now, thanks!